pkg/kopia/cli/internal/args/args_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package args_test
+
+import (
+	"testing"
+
+	"github.com/kanisterio/kanister/pkg/kopia/cli"
+	"github.com/kanisterio/kanister/pkg/kopia/cli/internal/args"
+	"github.com/kanisterio/safecli/test"
+	"gopkg.in/check.v1"
+)
+
+func TestArgs(t *testing.T) { check.TestingT(t) }
+
+var _ = check.Suite(&test.ArgumentSuite{Cmd: "cmd", Arguments: []test.ArgumentTest{
+	{
+		Name:        "Invalid ID",
+		Argument:    args.ID(""),
+		ExpectedErr: cli.ErrInvalidID,
+	},
+	{
+		Name:        "ID",
+		Argument:    args.ID("id12345"),
+		ExpectedCLI: []string{"cmd", "id12345"},
+	},
+}})
```

Comment:
Not test to run defined? Am I missing something?

Reply:
What do you mean? We have 2 tests here:

```
func TestArgs(t *testing.T) { check.TestingT(t) }
// Go test/bridge func for `go test` and delegate the actual testing work to the `check` framework.

// register `test.ArgumentSuite` with the `check` framework.
var _ = check.Suite(&test.ArgumentSuite{Cmd: "cmd", Arguments: []test.ArgumentTest{
	{
		Name:        "Invalid ID",
		Argument:    args.ID(""),
		ExpectedErr: cli.ErrInvalidID,
	},
	{
		Name:        "ID",
		Argument:    args.ID("id12345"),
		ExpectedCLI: []string{"cmd", "id12345"},
	},
}})
```

We're using [`github.com/kanisterio/safecli/test.ArgumentSuite`](https://github.com/kanisterio/safecli/blob/main/test/arg_suite.go#L28) to run table tests.